packages/nocodb/src/services/v3/filters-v3.service.ts

```
@@ -0,0 +1,190 @@
+      ...remappedFilter,
+      fk_view_id: param.viewId,
+    });
+
+    this.appHooksService.emit(AppEvents.FILTER_CREATE, {
+      filter,
+      view,
+      req: param.req,
+    });
+
+    return filterBuilder().build(filter);
+  }
```

Biome (1.9.4)

[error] Duplicate class member name "filterCreate"



(lint/suspicious/noDuplicateClassMembers)